### ISSUE TITLE
How to change the menu order of the pages

### ISSUE THREAD

**Issue by parijke** [Coderberg/ResidenceCMS#7]
If I create a new page I would like to place it in the menu where I like. Now it seems it has some default order.

**Comment by parijke** [comment]
It is ordered alphabetically I think
```
    public function findMenuItems()
    {
        return $this->createQueryBuilder('p')
            ->select('p.title', 'p.slug')
            ->andWhere('p.show_in_menu = 1')
            ->orderBy('p.title', 'ASC')
            ->getQuery()
            ->getResult();
    }
```

Maybe it is an option to add a weigth paramater to order the menu-items.

**Comment by Coderberg** [comment]
Menu editor is implemented in version 1.4.0
